packages/types/src/scripts/MetadataMd.ts

```diff
@@ -40,18 +42,40 @@ function addRpc () {
   }, renderHeading);
 }
 
+function sortExtrinsicsSectionMethods () {
+  return function (a: any, b: any) {
+    const nameA = a.raw.name.raw.toUpperCase(); // ignore upper and lowercase
+    const nameB = b.raw.name.raw.toUpperCase(); // ignore upper and lowercase
+
+    if (nameA < nameB) {
```

Comment:
Here we should just be able to do `nameA.localaCompare(nameB)` (conveted to strings above)